src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,61 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Mark Command, in charge of marking a task as done/undone.
+ */
+public class MarkCommand extends Command {
+    public enum Mark {
+        MARK,
+        UNMARK;
+    }
```

Comment:
I think it is an interesting idea to use an enum to differentiate between mark and unmarking commands! I personally had 2 classes (`MarkCommand.java` and `UnmarkCommand.java`) which resulted in a lot of repetition.